demo/index.html

```diff
@@ -141,7 +141,7 @@ <h1 class="mdl-mega-footer__heading" localized-string="PROJECT_LINKS_HEADER"></h
               <ul class="mdl-mega-footer__link-list">
                 <li><a rel="noopener" target="_blank" href="../docs/api/index.html" localized-string="DOCUMENTATION"></a></li>
                 <li><a rel="noopener" target="_blank" href="https://www.apache.org/licenses/LICENSE-2.0" localized-string="LICENSE"></a></li>
-                <li><a rel="noopener" target="_blank" href="https://github.com/google/shaka-player" localized-string="SOURCE"></a></li>
+                <li><a rel="noopener" target="_blank" href="https://github.com/shaka-project/shaka-player" localized-string="SOURCE"></a></li>
                 <li><a rel="noopener" target="_blank" href="https://www.npmjs.com/package/shaka-player" localized-string="NPM"></a></li>
```

Comment:
Speaking of NPM, do we need to update the links on the NPM page?

Reply:
No, because that isn't namespaced the same way our GitHub repos are.